cumulusci/cumulusci.yml

```diff
@@ -502,6 +502,18 @@ flows:
                     publish: True
                     tag: ^^github_release.tag_name
 
+    retrieve_scratch:
+        description: Retrieves declarative changes made in a scratch org and converts to src directory
+        steps:
+            1:
+                task: dx_convert_to
+            2:
+                task: dx_pull
+            3:
+                task: dx_convert_from
+            4:
+                task: update_package_xml
```

Comment:
This was meant for a different branch, right?

Reply:
yes, it's in a separate PR which should be merged before this PR and would cause this one to match master.  I accidentally pushed to this branch :)

Reply:
After merging the other PR and merging master into this branch, the diff is now gone